src/main/java/org/folio/rest/impl/DocumentHelper.java

```diff
@@ -0,0 +1,69 @@
+package org.folio.rest.impl;
+
+import static org.folio.invoices.utils.HelperUtils.buildQuery;
+import static org.folio.invoices.utils.HelperUtils.handleDeleteRequest;
+import static org.folio.invoices.utils.HelperUtils.handleGetRequest;
+import static org.folio.invoices.utils.ResourcePathResolver.*;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.invoices.utils.ResourcePathResolver;
+import org.folio.rest.jaxrs.model.DocumentCollection;
+import org.folio.rest.jaxrs.model.InvoiceDocument;
+
+import io.vertx.core.Context;
+import io.vertx.core.json.JsonObject;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+
+class DocumentHelper extends AbstractHelper {
+  private static final String GET_DOCUMENTS_BY_QUERY = resourcesPath(INVOICE_DOCUMENTS) + "?limit=%s&offset=%s%s&lang=%s";
+
+  DocumentHelper(Map<String, String> okapiHeaders, Context ctx, String lang) {
+    super(getHttpClient(okapiHeaders), okapiHeaders, ctx, lang);
+  }
+
+  CompletableFuture<InvoiceDocument> createDocument(String invoiceId, InvoiceDocument document) {
+    JsonObject jsonDocument = JsonObject.mapFrom(document);
+    String endpoint = String.format(resourcesPath(ResourcePathResolver.INVOICE_DOCUMENTS), invoiceId);
+    return createRecordInStorage(jsonDocument, endpoint).thenApply(id -> {
+      document.getDocumentMetadata().setId(id);
+      return document;
+    });
+  }
+
+  CompletableFuture<DocumentCollection> getDocumentsByInvoiceId(String invoiceId, int limit, int offset, String query) {
+    CompletableFuture<DocumentCollection> future = new VertxCompletableFuture<>(ctx);
+    String queryParam = buildQuery(query, logger);
```

Comment:
Please use `HelperUtils.getEndpointWithQuery` instead. `buildQuery` method is duplicate of that one and I am going to remove it in https://github.com/folio-org/mod-invoice/pull/55